src/neo/SmartContract/Manifest/ContractManifest.cs

```diff
@@ -117,7 +117,9 @@ public static ContractManifest FromJson(JObject json)
         /// </summary>
         /// <param name="json">Json</param>
         /// <returns>Return ContractManifest</returns>
-        public static ContractManifest Parse(string json) => FromJson(JObject.Parse(json));
+        public static ContractManifest Parse(ReadOnlySpan<byte> json) => FromJson(JObject.Parse(json));
+
+        internal static ContractManifest Parse(string json) => FromJson(JObject.Parse(json));
```

Comment:
should be public?

https://github.com/neo-project/neo-node/blob/004a3b7ba7744b9d9d3dd2a6e2e652068282a28f/neo-cli/CLI/MainService.cs#L248

Reply:
It is recommended to use this overload.

https://github.com/neo-project/neo/blob/063357ed762257c30455103c95501949445c6039/src/neo/SmartContract/Manifest/ContractManifest.cs#L120